wagtail/wagtailimages/templates/wagtailimages/multiple/add.html

```diff
@@ -22,6 +22,18 @@
                     <input id="fileupload" type="file" name="files[]" data-url="{% url 'wagtailimages:add_multiple' %}" multiple>
                 </div>
                 {% csrf_token %}
+                {% if collections %}
+                    <div class="field">
+                        <label for="id_addimage_collection">Add to collection:</label>
```

Comment:
Would be great to mark this string for translation. The same in `wagtail/wagtaildocs/templates/wagtaildocs/multiple/add.html`